internal/provider/data_source_cloudexport_item_test.go

```diff
@@ -183,3 +189,306 @@ func makeTestCloudExportDataSourceItems(apiURL string) string {
 		apiURL,
 	)
 }
+
+func TestAccDataSourceCloudExportItemAWS(t *testing.T) {
+	ce, err := createTestAccCloudExportItemAWS()
```

Comment:
What happens in this function when `TF_ACC` is not set in the environment? Is anything tested, or is the run totally useless?

Reply:
If TF_ACC is not set, any test functions whose name starts with TestAccXxx will not be executed.

Reply:
Is that guaranteed by the Terraform test framework? If so, why do we need the `TC_ACC` checks in `{make,create}TestAccXxx` functions? AFAICT, those functions are called only from `TestAccxxx` functions.

Reply:
If TF_ACC variable is not set, terraform test framework will not run TestCase() in TestAcc*** functions, but will run any other code not placed in TestCase(). So I added skipIfNotAcceptance() to prevent it from running during normal tests.